.eslintrc.json

```diff
@@ -23,7 +23,7 @@
     "no-undef": 2,
     "no-use-before-define": [2, "nofunc"],
     "no-unused-expressions": [2, { "allowShortCircuit": true, "allowTernary": true }],
-    "no-unused-vars": [2, { "args": "none" }],
```

Comment:
not sure the back story but I think there is no reason to disable rule for unused arguments?